rrfs-test/IODA/yaml/prepbufr_aircft.yaml

```diff
@@ -31,112 +29,110 @@ observations:
             query: "*/YOB"
           stationIdentification:
             query: "*/SID"
-          aircraftFlightNumber:
-            query: "[*/ACID, */ACID_SEQ/ACID]"
-          prepbufrDataLevelCategory:
-            query: "*/PRSLEVLA/CAT"
+          stationElevation:
+            query: "*/ELV"
+            type: float
           prepbufrReportType:
             query: "*/TYP"
           dumpReportType:
             query: "*/T29"
           prepbufrDataLvlCat:
-            query: "*/PRSLEVLA/CAT"
+            query: "*/PRSLEVLA{1}/CAT"
           aircraftPhase:
-            query: "*/PRSLEVLA/ACFT_SEQ/POAF"
-          stationElevation:
-            query: "*/ELV"
-            type: float
+            query: "*/PRSLEVLA{1}/ACFT_SEQ/POAF"
+          aircraftFlightNumber:
+            query: "[*/ACID, */ACID_SEQ/ACID]"
 
           pressure:
-            query: "*/PRSLEVLA/P___INFO/P__EVENT{1}/POB"
+            query: "*/PRSLEVLA{1}/P___INFO/P__EVENT{1}/POB"
```

Comment:
Sorry I forgot about this. @delippi could you remind me what the `{1}` does?

Reply:
Without the`{1}` I was getting POBs with two dimensions `(Location, Level)` where the dimension of Level was 1. This addition just makes all the data like `airTemperature(Location)` instead of `airTemperature(Location, Level)`.

Reply:
Thanks for the reiteration on this.